server/services/database/message.js

```diff
@@ -11,7 +11,10 @@ const createMessage = ({ recruiter_id, profile_public_id, message }) => {
 
 const getPublicIdsByRecruiterId = (id) => {
```

Comment:
It is a good time now to change this function name because now it is getting more content, it could be something like `getMessagesByRecruiterId` as it was before.

Reply:
Changed. Thanks for your assistance.